mne/preprocessing/ica.py

```diff
@@ -440,16 +455,18 @@ def _pre_whiten(self, data, info, picks):
             # Scale (z-score) the data by channel type
             info = pick_info(info, picks)
             pre_whitener = np.empty([len(data), 1])
-            for ch_type in _DATA_CH_TYPES_SPLIT:
+            for ch_type in _DATA_CH_TYPES_SPLIT + ['eog']:
                 if _contains_ch_type(info, ch_type):
                     if ch_type == 'seeg':
                         this_picks = pick_types(info, meg=False, seeg=True)
                     elif ch_type == 'ecog':
                         this_picks = pick_types(info, meg=False, ecog=True)
                     elif ch_type == 'eeg':
                         this_picks = pick_types(info, meg=False, eeg=True)
-                    else:
+                    elif ch_type in ['mag', 'grad']:
                         this_picks = pick_types(info, meg=ch_type)
+                    elif ch_type == 'eog':
+                        this_picks = pick_types(info, meg=False, eog=True)
```

Comment:
The code catches ref_meg and says it is not a supported channel. Is that the desired behaviour? @Eric89GXL @agramfort @jaeilepp